plugins/UFPReader/plugin.json

```diff
@@ -3,6 +3,6 @@
     "author": "Ultimaker B.V.",
     "version": "1.0.0",
     "description": "Provides support for reading Ultimaker Format Packages.",
-    "supported_sdk_versions": ["6.0.0"],
+    "supported_sdk_versions": ["7.0.0"],
```

Comment:
To keep it consistent with the others, I'd change this to `7.0`

Reply:
Same as above, 7.0.0 is correct semver, 7.0 (even though we accept it) is technically not.